apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```diff
@@ -123,6 +137,27 @@ $tokensTable-item-vertical-padding: f.spacing(e);
     }
 }
 
+.c-tokensTable-example--elevation {
+    background: var(--example-elevation);
+    width: 64px;
+    height: 64px;
+    border-radius: f.$radius-03;
+    margin: 0 auto;
+    left: 0;
+    right: 0;
+    position: absolute;
+    bottom: 40px;
+    top: 56px;
+    box-shadow: var(--example-shadow);
+
+    @include f.media('>=wide') {
+        bottom: 24px;
+        top: 24px;
+        width: 32px;
+        height: 32px;
+    }
+}
+
 .c-tokensTable-example-radius {
```

Comment:
nitpick: I know you didn't add this as a part of the PR but `c-tokensTable-example-radius` probably needs to be `c-tokensTable-example--radius` - would you mind changing this as part of your PR please?